views/secretWord.ejs

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
```

Comment:
Not a blocker on this assignment, but all of this should be in the `head.ejs` partial